output/cloud/expv2/metrics_client.go

```diff
@@ -44,15 +41,17 @@ type metricsClient struct {
 }
 
 // newMetricsClient creates and initializes a new MetricsClient.
-func newMetricsClient(logger logrus.FieldLogger, host string, token string) (*metricsClient, error) {
+func newMetricsClient(
+	logger logrus.FieldLogger, host string, token string, timeout time.Duration,
+) (*metricsClient, error) {
 	if host == "" {
 		return nil, errors.New("host is required")
 	}
 	if token == "" {
 		return nil, errors.New("token is required")
 	}
 	return &metricsClient{
-		httpClient: &http.Client{Timeout: 5 * time.Second},
+		httpClient: &http.Client{Timeout: timeout},
```

Comment:
Is there a particular reason you can't use the clouadpi client?

It does all of the relevant parts and already sets this timeout. 

I am also pretty sure we have an instance in the wrapping `output` struct that is used to create the test run if needed.

Reply:
The main reason was the JSON payload + headers but checking better I've found a way for work-around them.